src/pharmpy/modeling/plots.py

```diff
@@ -211,7 +211,13 @@ def plot_transformed_eta_distributions(
     return facet
 
 
-def plot_dv_vs_pred(model: Model, predictions: pd.DataFrame) -> alt.Chart:
+def plot_dv_vs_pred(
+    model: Model,
+    predictions: pd.DataFrame,
+    strat: str = None,
```

Comment:
I think we should use the full word here. Either `stratification` or `stratify_on` or perhaps `by`, which is what xpose4 is using.